src/sentry/utils/audit.py

```diff
@@ -146,13 +157,20 @@ def _complete_delete_log(delete_log: DeletedEntry, entry: AuditLogEntry) -> None
 
 
 def create_system_audit_entry(
-    transaction_id: int | str | None = None, logger: Logger | None = None, **kwargs: Any
+    transaction_id: int | str | None = None,
+    logger: Logger | None = None,
+    organization: Organization | None = None,
+    organization_id: int | None = None,
+    **kwargs: Any,
 ) -> AuditLogEntry:
     """
     Creates an audit log entry for events that are triggered by Sentry's
     systems and do not have an associated Sentry user as the "actor".
     """
-    entry = AuditLogEntry(actor_label="Sentry", **kwargs)
+    if organization:
+        organization_id = organization.id
```

Comment:
Same here with guarding against both `organization` and `organization_id` being provided together.